_activities/cwp.md

```diff
@@ -48,6 +49,8 @@ sub-domains. All of the reports produced during the Community White Paper proces
 
 This is a list of talks about the CWP Roadmap that were given by various HSF members:
 
+* 2018-01-10 [Presentation](https://indico.cern.ch/event/651349/contributions/2830237/attachments/1580497/2497360/cwp-gdb-january-2018.pdf) at the [January 2018 GDB](https://indico.cern.ch/event/651349/) by Graeme Stewart.
+* 2017-01-05 [Presentation](https://indico.cern.ch/event/666278/contributions/2830239/attachments/1579352/2495271/Advances_in_software_and_computing_for_HEP.pdf) at the [2018 Sp&aring;tlid Nordic Particle Physics Meeting](https://indico.cern.ch/event/666278/overview) by Graeme Stewart.
```

Comment:
I would add the locations in the 2nd links, for info. For example I do not know where the Nordic meeting happened ;-).